evm/src/cpu/kernel/tests/account_code.rs

```diff
@@ -369,26 +397,35 @@ fn sload() -> Result<()> {
 
     // Prepare the interpreter by inserting the account in the state trie.
     prepare_interpreter_all_accounts(&mut interpreter, trie_inputs, addr, &code)?;
-
     interpreter.run()?;
 
     // The first two elements in the stack are `success` and `leftover_gas`,
     // returned by the `sys_stop` opcode.
-    interpreter.pop();
-    interpreter.pop();
+    interpreter
+        .pop()
+        .expect("The stack length should not be empty.");
+    interpreter
+        .pop()
+        .expect("The stack length should not be empty.");
 
     // The SLOAD in the provided code should return 0, since
     // the storage trie is empty. The last step in the code
     // pushes the value 3.
     assert_eq!(interpreter.stack(), vec![0x0.into(), 0x3.into()]);
-    interpreter.pop();
-    interpreter.pop();
+    interpreter
+        .pop()
+        .expect("The stack length should not be empty.");
+    interpreter
+        .pop()
+        .expect("The stack length should not be empty.");
     // Now, execute mpt_hash_state_trie. We check that the state trie has not changed.
     let mpt_hash_state_trie = KERNEL.global_labels["mpt_hash_state_trie"];
     interpreter.generation_state.registers.program_counter = mpt_hash_state_trie;
     interpreter.set_is_kernel(true);
     interpreter.set_context(0);
-    interpreter.push(0xDEADBEEFu32.into());
+    interpreter
+        .push(0xDEADBEEFu32.into())
+        .expect("Stack should not overflow.");
```

Comment:
nit: Maybe rather 
```suggestion
        .expect("The stack should not overflow.");
```
to match `expect` messages above?